metadata-ingestion/src/datahub/ingestion/run/pipeline.py

```diff
@@ -512,7 +503,7 @@ def pretty_print_summary(
             )
             num_failures_sink = len(self.sink.get_report().failures)
             click.secho(
-                f"{'⏳' if currently_running else ''} Pipeline {'running' if currently_running else 'finished'} with at least {num_failures_source+num_failures_sink} failures {'so far' if currently_running else ''}; produced {workunits_produced} events {duration_message}",
+                f"{'⏳' if currently_running else ''} Pipeline {'running' if currently_running else 'finished'} with at least {num_failures_source+num_failures_sink} failures{' so far' if currently_running else ''}; produced {workunits_produced} events {duration_message}",
```

Comment:
is this change intentional? cannot tell

Reply:
yup it was intentional - it removes the space from the print of `failures ;`